scenarios/omoo_system.json

```diff
@@ -65,7 +64,7 @@
                 "use_smartds": false,
                 "profile_location": "gadal_ieee123/profiles",
                 "opendss_location": "gadal_ieee123/qsts",
-                "sensor_location": "gadal_ieee123/sensors.json",
+                "sensor_location": "sensors.json",
```

Comment:
Why is this location modified by the opendss and profile names still include the gadal_iee123 name?

Reply:
Ok I figured out what was happening here. The problem is this existing_feeder_file just out of frame.

We use the `sensor_location` in two different ways right now. In one way, we specify where the sensors are _on_ AWS. In the other way, we use it put it on the channel. I was using it for both, which is why is became a problem.